src/garage/experiment/experiment.py

```diff
@@ -240,11 +239,8 @@ def run_experiment(method_call=None,
             commands without executing them.
         env (dict): Extra environment variables.
         variant (dict): If provided, should be a dictionary of parameters.
-        use_tf (bool): Used along with the Theano and GPU configuration
-            when using TensorFlow
         use_gpu (bool): Whether the launched task is running on GPU.
```

Comment:
Considering this flag's implementation, I actually think `use_gpu` is a pretty bad name. It doesn't ensure that the GPU is used for tensorflow or pytorch. For tensorflow, even if you set `use_gpu`, the gpu is only actually used if you have tensorflow-gpu installed. For pytorch, the gpu currently doesn't get used unless you manually move the policy to the gpu (by calling `.to(device)` or `.cuda()` on it).

Perhaps we should instead have a flag called `force_cpu`, which defaults to `False`?